.pre-commit-config.yaml

```diff
@@ -28,8 +28,8 @@ repos:
     rev: v0.11.7
     hooks:
       - id: ruff
```

Comment:
https://github.com/vllm-project/vllm/pull/26255  I think we can follow the example of the vllm pull request and update the Ruff Hook version.